src/server.js

```diff
@@ -80,6 +80,21 @@ Server.prototype.writeServerConfig = function () {
   fs.writeFileSync(file, config)
 }
 
+Server.prototype.writeBattleEyeConfig = function (config) {
+  if(this.options.battleEye)
```

Comment:
Does this need to be conditional? Would the function be called if it isn't desired to create the file?